src/it/java/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/GenericWhitespaceTest.java

```diff
@@ -42,22 +42,22 @@ public void testWhitespaceAroundGenerics() throws Exception {
         final Map<String, String> messages = checkConfig.getMessages();
 
         final String[] expected = {
-            "12:16: " + getCheckMessage(messages, msgPreceded, "<"),
-            "12:18: " + getCheckMessage(messages, msgFollowed, "<"),
-            "12:36: " + getCheckMessage(messages, msgPreceded, "<"),
-            "12:38: " + getCheckMessage(messages, msgFollowed, "<"),
-            "12:47: " + getCheckMessage(messages, msgPreceded, ">"),
-            "12:49: " + getCheckMessage(messages, msgFollowed, ">"),
-            "12:49: " + getCheckMessage(messages, msgPreceded, ">"),
-            "14:32: " + getCheckMessage(messages, msgPreceded, "<"),
-            "14:34: " + getCheckMessage(messages, msgFollowed, "<"),
-            "14:45: " + getCheckMessage(messages, msgPreceded, ">"),
-            "15:32: " + getCheckMessage(messages, msgPreceded, "<"),
-            "15:34: " + getCheckMessage(messages, msgFollowed, "<"),
-            "15:45: " + getCheckMessage(messages, msgPreceded, ">"),
-            "20:38: " + getCheckMessage(messages, msgPreceded, "<"),
-            "20:40: " + getCheckMessage(messages, msgFollowed, "<"),
-            "20:61: " + getCheckMessage(messages, msgPreceded, ">"),
+            "12:17: " + getCheckMessage(messages, msgFollowed, "<"),
```

Comment:
+1 is correct, verified at InputWhitespaceAroundGenerics, symbol `<` is at 17 column.